samples/cpp/module_genai/ut_modules/ZImageDenoiserLoopModule.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "../ut_modules_base.hpp"
+
+class ZImageDenoiserLoopModule : public ModuleTestBase {
+public:
+    DEFINE_MODULE_TEST_CONSTRUCTOR(ZImageDenoiserLoopModule)
+
+protected:
+    std::string get_yaml_content() override {
+        return R"(
+global_context:
+  model_type: "zimage"
+pipeline_modules:
+
+  denoiser_loop:
+    type: "ZImageDenoiserLoopModule"
+    device: "CPU"
+    inputs:
+      - name: "prompt_embed"
+        type: "OVTensor"
+        source: "pipeline_params.prompt_embed"
+      - name: "num_inference_steps"
+        type: "Int"
+        source: "pipeline_params.num_inference_steps"
+      - name: "width"
+        type: "Int"
+        source: "pipeline_params.width"
+      - name: "height"
+        type: "Int"
+        source: "pipeline_params.height"
+    outputs:
+      - name: "latent"
+        type: "OVTensor"
+    params:
+      model_path: "./ut_pipelines/Z-Image-Turbo-fp16-ov/"
+)";
+    }
+
+    ov::AnyMap prepare_inputs() override {
+        ov::AnyMap inputs;
+
+        auto prompt_embed = ut_randn_tensor(ov::Shape{101, 2560}, 42);
+        inputs["prompt_embed"] = prompt_embed;
+        inputs["num_inference_steps"] = 5;
```

Comment:
```suggestion
        inputs["num_inference_steps"] = 2;
```
Just reduce unit time and cover multiple inferences.